### PR TITLE
Add tests for disconnection with no nexus IO

### PR DESCRIPTION
**Pull request by chriswldenyer**
Disconnects a replica but sends no IO to it. Verifies that the the volume still becomes degraded and can be used for IO after this happens.
Shortened the directory names for the individual tests.
Made changes to match recent changes in the MSV spec.

### CODE REVIEW

File: mayastor-test/e2e/node_disconnect/lib/node_disconnect_lib.go [hunk 1]
```diff
@@ -97,6 +98,34 @@ func LossTest(nodeToIsolate string, otherNodes []string, disconnectionMethod str
...
 }
 
+// Remove the replica without running IO and verify that the volume becomes degraded but is still functional
+func LossWhenIdleTest(nodeToIsolate string, otherNodes []string, disconnectionMethod string, uuid string) {
+	fmt.Printf("disconnecting \"%s\"\n", nodeToIsolate)
+
+	DisconnectNode(nodeToIsolate, otherNodes, disconnectionMethod)
+
+	fmt.Printf("waiting up to 90s for disconnection to affect the nexus\n")
+	//time.Sleep(90 * time.Second)
```
Comment (paulyoong) [inline mayastor-test/e2e/node_disconnect/lib/node_disconnect_lib.go:108]: I think we can remove the commented out line and the print statement.

**Review by paulyoong — APPROVED** [review]

**Review by blaisedias — APPROVED** [review]

File: mayastor-test/e2e/node_disconnect/lib/node_disconnect_lib.go (hunk 1 above)
Comment (chriswldenyer) [reply mayastor-test/e2e/node_disconnect/lib/node_disconnect_lib.go:108]: I'll shortly be having another PR, so I'll remove it then. Thanks.

**Comment by chriswldenyer** [comment]
bors merge

**Comment by chriswldenyer** [comment]
bors merge

**Pull request merged**